cypress/integration/dib-travel/company-settings/approval-process/helpers/select-approval-settings.ts

```diff
@@ -1,30 +1,22 @@
-import { getFirstWord } from '@cy/helpers';
-
 export const selectApprovalSettings = (settings: ApprovalSettings, groupName: string): void => {
   let inputSelectorPosition = -1;
-  let groupSelectorPosition = -1;
```

Comment:
Since you removed previous variations, you can replace this:

```
  let inputSelectorPosition = -1;

  switch (settings) {
    case 'Only out of policy travels':
      inputSelectorPosition = 1;
      break;
    case 'All travels':
      inputSelectorPosition = 1;
      break;
  }
  ```
  
just with `  let inputSelectorPosition = 1;`

Reply:
I made changes, but I only used a `const` instead of a `let`, because the typescript corrected me :)